nsds_lab_to_nwb/metadata/exp_note_reader.py

```diff
@@ -25,24 +25,28 @@ def __init__(self, path, block_folder):
         self.input_format = None
         self.block_folder = block_folder
         _, _, blockstr = split_block_folder(block_folder)
-        self.block_id = int(blockstr[1:])
-        
+        self.block_id = int(blockstr[1:])        
         self.file = []
+        self._raw_meta = None 
+        self._raw_block = None
         
         # autodetect input format
```

Comment:
How about also having an input kwarg that can force a specific input format? (set the default value for this kwarg to None, in which case autodetect is used?)

Reply:
(I meant to highlight the line "#autodetct input format")

Reply:
Added this feature.